community/examples/slurm-gcp-v5-hpc-centos7.yaml

```diff
@@ -18,7 +18,7 @@ blueprint_name: slurm-gcp-v5-hpc-centos7
 
 vars:
   project_id:  ## Set GCP Project ID Here ##
-  deployment_name: slurm-gcp-v5
```

Comment:
V5 pertains to slurm-gcp version, so I think your changes make it a little confusing. What about moving the distro to the beginning?  eg: 
centos-slurm-gcp-v5
ubuntu-slurm-gcp-v5

Reply:
> V5 pertains to slurm-gcp version, so I think your changes make it a little confusing. What about moving the distro to the beginning? eg: centos-slurm-gcp-v5 ubuntu-slurm-gcp-v5

Make sense. I would like to keep at least the "slurm" after name get shorted but I don't like the idea of abbreviate distro. I can't think other possibilities given the scenario. If you have any, will be welcome otherwise let go with them.

Reply:
Ok, so I talked with the team internally and we believe there is no good solution here. We will talk to SchedMD and try to remove this limitation of 10 characters in the future. 
Let's close this PR now and accept that these two deployments will temporarily clash until we fix this in a better way. 
What do you think?